README.md

```diff
@@ -90,6 +90,9 @@ Finally, to run the api locally:
 npm run dev
 ```
 
+- View the documentation locally through port 3000
```

Comment:
Maybe we can say localhost:3000 to keep it consistent with the previous localhost port